src/renderer/component/publishForm/view.jsx

```diff
@@ -1,6 +1,6 @@
 // @flow
 import * as React from 'react';
-import { isNameValid, buildURI, regexInvalidURI } from 'lbry-redux';
+import { isNameValid, buildURI, regexInvalidURI, STATUSES } from 'lbry-redux';
```

Comment:
what kind of statuses are these?

Reply:
spee.ch api status'. Agreed we should probably rename.